parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -94,6 +95,8 @@ KUBECTL=/usr/local/bin/kubectl
 DOCKER=/usr/bin/docker
 export GPU_DV="{{GPUDriverVersion}}"
 export GPU_DEST=/usr/local/nvidia
+MOBY_VERSION=20.10.15
```

Comment:
I believe this is where we should be doing the docker bump.

Reply:
yeah maybe let's just bump to 20.10.17 if that works for all distros?